app/assets/javascripts/discourse/app/components/modal/reject-reason-reviewable.js

```diff
@@ -0,0 +1,17 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+
+export default class RejectReasonReviewableModal extends Component {
+  rejectReason;
+  sendEmail = false;
```

Comment:
I'm kind of surprised both of these don't need `@tracked` for the data binding...maybe just double check `perform()` has the right values. I could be totally wrong, it just stands out to me :)

Reply:
Nice, I am wrong on Ember stuff yet again, I knew it :laughing: